<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIForm</class>
 <widget class="QDialog" name="SIForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Surgeon's GUI for a BRL's Laparoscopic Robot System</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>570</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Surgeon</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_DataCollection">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>395</y>
       <width>330</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="title">
      <string>Data Connection</string>
     </property>
     <widget class="QPushButton" name="DC_saveButton">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>23</y>
        <width>100</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DC_dictateButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>23</y>
        <width>100</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Dictate Note</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DC_newButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>23</y>
        <width>100</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Generate
New File</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="buttonGroup_View">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>260</y>
       <width>330</width>
       <height>130</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="title">
      <string>Video Source Select </string>
     </property>
     <widget class="QRadioButton" name="RB_View_Endoscope">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>24</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Endoscopic View</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RB_View_OR2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>94</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Operating Room View 2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RB_View_OR">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>59</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Operating Room View 1</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Scale">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>155</y>
       <width>330</width>
       <height>105</height>
      </rect>
     </property>
     <property name="title">
      <string>Scale</string>
     </property>
     <widget class="QPushButton" name="scaleDecreaseButton">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>28</y>
        <width>100</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="scaleIncreaseButton">
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>28</y>
        <width>100</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="scaleLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>33</y>
        <width>110</width>
        <height>61</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="okButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>40</y>
       <width>160</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>CLOSE GUI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectionButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>190</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Connection Status</string>
     </property>
     <property name="iconSize">
      <size>
       <width>190</width>
       <height>100</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>154</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Left Hand Tool</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>171</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Right Hand Tool</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>213</x>
       <y>130</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Elapsed Surgery Time</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>220</y>
       <width>390</width>
       <height>280</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>390</width>
       <height>280</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Time">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>160</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Now Logined
Press to log-out</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>330</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>bioRobotics Laboratory</string>
     </property>
     <property name="iconSize">
      <size>
       <width>330</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cB_LeftTool">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>49</y>
       <width>180</width>
       <height>51</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="cB_RightTool">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>49</y>
       <width>180</width>
       <height>51</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>Engineer</string>
    </attribute>
    <widget class="QGroupBox" name="buttonGroup_CM">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>360</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="title">
      <string>Select Control Mode</string>
     </property>
     <widget class="QRadioButton" name="rB_CM_1D">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>36</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>One Directional Control</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rB_CM_PP">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>68</y>
        <width>271</width>
        <height>17</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Position-Position Control</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Address</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lCDNumber2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>160</width>
       <height>35</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="A_okButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>160</width>
       <height>70</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>CLOSE GUI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="A_startButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>110</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>START</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cB_IP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="infoText">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>10</y>
       <width>390</width>
       <height>510</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Theta 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Theta 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Theta 3</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="camAngle1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>350</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="camAngle2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>380</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="camAngle3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>410</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="ITPCheck">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Use ITP Reference Frame</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
